keyboards/planck/keymaps/dvorak2space/keymap.c

```diff
@@ -145,6 +146,11 @@ bool process_record_user(uint16_t keycode, keyrecord_t *record) { //X_KEY doesn'
     case HK_ELSE:
       if(record->event.pressed) { SEND_STRING("else"); }
       return false;
+    case HK_COSL:
+      reset_oneshot_layer();
+      if(IS_LAYER_ON(AHK_L)) { layer_invert(AHK_L); }
+      if(IS_LAYER_ON(PASS_L)) { layer_invert(PASS_L); }
+      return false;
```

Comment:
```suggestion
      return true;
```

Reply:
The point of that was to reset the OSL as if not in the tapping term tapping it again didn't close it, but I just realized returning true would do that anyway. Committing as empty with return true as well as changing other custom functions to return true.